src/ILLink.RoslynAnalyzer/ReflectionMethodBodyScanner.cs

```diff
@@ -91,6 +108,37 @@ public static bool HandleCall (OperationAnalysisContext context, IInvocationOper
 			var calledMethod = invocationOperation.TargetMethod;
 			switch (GetIntrinsicIdForMethod (calledMethod)) {
 
+				//
+				// GetMethod (string)
+				// GetMethod (string, BindingFlags)
+				// GetMethod (string, Type[])
+				// GetMethod (string, Type[], ParameterModifier[])
+				// GetMethod (string, BindingFlags, Type[])
+				// GetMethod (string, BindingFlags, Binder, Type[], ParameterModifier[])
+				// GetMethod (string, BindingFlags, Binder, CallingConventions, Type[], ParameterModifier[])
+				// GetMethod (string, int, Type[])
+				// GetMethod (string, int, Type[], ParameterModifier[]?)
+				// GetMethod (string, int, BindingFlags, Binder?, Type[], ParameterModifier[]?)
+				// GetMethod (string, int, BindingFlags, Binder?, CallingConventions, Type[], ParameterModifier[]?)
+				//
+				case IntrinsicId.Type_GetMethod: {
+						BindingFlags? bindingFlags;
+						// @TODO - only handling the first 2 cases here
+						if (calledMethod.Parameters.Length > 1 && calledMethod.Parameters[1].Type.GetDisplayName () == "System.Reflection.BindingFlags")
+							bindingFlags = GetBindingFlagsFromValue (invocationOperation.Arguments[1]);
+						else
+							// Assume a default value for BindingFlags for methods that don't use BindingFlags as a parameter
+							bindingFlags = BindingFlags.Public;
```

Comment:
Since the binder for intrinsics operates with BindingFlags the default BindingFlags for GetMethod are BindingFlags.Instance | BindingFlags.Static | BindingFlags.Public